### PR TITLE
[plog] Update to 1.1.6

### PR DESCRIPTION
**Pull request by chausner**
**Describe the pull request**

- #### What does your PR fix?  
  Updates plog to 1.1.6

- #### Which triplets are supported/not supported? Have you updated the [CI baseline](https://github.com/microsoft/vcpkg/blob/master/scripts/ci.baseline.txt)?  
  As before, Yes

- #### Does your PR follow the [maintainer guide](https://github.com/microsoft/vcpkg/blob/master/docs/maintainers/maintainer-guide.md)?  
  Yes

- #### If you have added/updated a port: Have you run ./vcpkg x-add-version --all and committed the result?  
  Yes

### CODE REVIEW

File: ports/plog/portfile.cmake [hunk 1]
```diff
@@ -2,14 +2,17 @@
...
+    REF 1.1.6
+    SHA512 5240532df96144d5026d6f879b69f7d6f393ebc9122c6458c41076fd3db998565e45ed2ab4948f8c3cb59e08c0aad7695ee416f95b49fd70209cd937220cdf8b
     HEAD_REF master
 )
 
 # Put the licence file where vcpkg expects it
-file(COPY ${SOURCE_PATH}/LICENSE DESTINATION ${CURRENT_PACKAGES_DIR}/share/plog)
-file(RENAME ${CURRENT_PACKAGES_DIR}/share/plog/LICENSE ${CURRENT_PACKAGES_DIR}/share/plog/copyright)
+file(COPY "${SOURCE_PATH}/LICENSE" DESTINATION "${CURRENT_PACKAGES_DIR}/share/plog")
+file(RENAME "${CURRENT_PACKAGES_DIR}/share/plog/LICENSE" "${CURRENT_PACKAGES_DIR}/share/plog/copyright")
```
Comment (Thomas1664) [inline ports/plog/portfile.cmake:12]:
```suggestion
file(INSTALL "${SOURCE_PATH}/LICENSE" DESTINATION "${CURRENT_PACKAGES_DIR}/share/${PORT}" RENAME copyright)
```
+ Please put this at the end after copying usage

File: ports/plog/usage [hunk 2]
```diff
@@ -0,0 +1,4 @@
+The package plog is header only and can be used from CMake via:
+
+    find_path(PLOG_INCLUDE_DIRS "plog/Log.h")
+    target_include_directories(main PRIVATE ${PLOG_INCLUDE_DIRS})
```
Comment (Thomas1664) [inline ports/plog/usage:4]:
```suggestion
    target_include_directories(main PRIVATE ${PLOG_INCLUDE_DIRS})

```
Please add a final newline

**Review by Thomas1664 — CHANGES_REQUESTED** [review]

File: ports/plog/portfile.cmake (hunk 1 above)
Comment (chausner) [reply ports/plog/portfile.cmake]: Done in 774409038e679889cea487c7d1c67a7d39c28842.

File: ports/plog/usage (hunk 2 above)
Comment (chausner) [reply ports/plog/usage]: Done in 774409038e679889cea487c7d1c67a7d39c28842

**Comment by strega-nil-ms** [comment]
Thanks!

**Pull request merged**